src/online/verifier.rs

```diff
@@ -270,6 +273,7 @@ impl<D: Domain, PI: Iterator<Item = Instruction<D::Scalar>>> StreamingVerifier<D
         }
 
         if self.proof.runs.len() != D::ONLINE_REPETITIONS {
+            eprintln!("Length mismatch!");
```

Comment:
Nit: we might want to use `log` and `env_logger` here instead of `eprintln!`.

Alternatively: if matching these indicates that things are irrecoverably broken, we could use `assert!` or `debug_assert!` or make this function return a `Result` instead of an `Option`. Thoughts?

Reply:
Okay, I refactored it to return a Result, but I use a `std::String` as the error type, which might not be very idiomatic either? All I can really say is that it makes the code look kind of ugly.

Reply:
`Result<String>` is decently idiomatic, but ideally we'd have an `error.rs` module in Reverie and define our own `Error` type that `impl`s `std::error::Error`. But we can do that in a follow-up 😄